src/Workspaces/CSharp/Portable/Classification/Worker_DocumentationComments.cs

```diff
@@ -158,13 +159,31 @@ private void ClassifyXmlTextToken(SyntaxToken token)
             }
             else if (token.Kind() != SyntaxKind.XmlTextLiteralNewLineToken)
             {
-                RoslynDebug.Assert(token.Parent is object);
+                RoslynDebug.Assert(token.Parent is not null);
                 switch (token.Parent.Kind())
                 {
                     case SyntaxKind.XmlText:
                         AddClassification(token, ClassificationTypeNames.XmlDocCommentText);
                         break;
                     case SyntaxKind.XmlTextAttribute:
+                        if (token.Parent is XmlTextAttributeSyntax { Name.LocalName.Text: "langword" })
```

Comment:
📝 I would prefer to see a new method `ClassifyXmlLangwordAttribute` called instead of `ClassifyXmlTextToken` for the `langword` attribute.